src/Core/Services/IOrganizationDomainService.cs

```diff
@@ -4,4 +4,8 @@ public interface IOrganizationDomainService
 {
     Task ValidateOrganizationsDomainAsync();
     Task OrganizationDomainMaintenanceAsync();
+    /// <summary>
+    /// Indicates if the organization has any verified domains.
+    /// </summary>
+    Task<bool> HasVerifiedDomainsAsync(Guid orgId);
```

Comment:
Should `OrganizationDomainService` be in the `AdminConsole` folder for code ownership purposes? We can move it without changing the namespace to avoid breaking imports if there are a lot of them, but while we're here it makes sense to move it if this a part of ACs domain.

Reply:
I moved the namespace, there were only two references.